docs/spark-standalone.md

```diff
@@ -240,6 +240,16 @@ SPARK_MASTER_OPTS supports the following system properties:
   </td>
   <td>1.6.3</td>
 </tr>
+<tr>
+  <td><code>spark.submit.waitAppCompletion</code></td>
+  <td><code>true</code></td>
```

Comment:
Default is false right?

Reply:
Yes, corrected it in the latest commit. Thanks for catching it.